lib/webpacker/file_loader.rb

```diff
@@ -3,10 +3,16 @@ class Webpacker::FileLoader
   class NotFoundError < StandardError; end
   class FileLoaderError < StandardError; end
 
-  class_attribute :instance
+  class_attribute :instance, :reloadable
   attr_accessor :data
 
   class << self
+    reloadable = false
```

Comment:
Should not this be `self.reloadable = false`? Otherwise it will only define a local variable.